tests/installation/encrypted_volume_activation.pm

```diff
@@ -40,10 +41,17 @@ sub run {
         }
     }
     elsif (get_var('ENCRYPT_ACTIVATE_EXISTING')) {
-        send_key 'alt-p';
-        assert_screen 'encrypted_volume_password_prompt';
-        type_password;
-        send_key 'ret';
+        # pre storage NG has an additional question dialog
+        if (match_has_tag 'encrypted_volume_activation_prompt-embedded_password_prompt') {
+            type_password;
+            send_key $cmd{ok};
+        }
+        else {
+            send_key 'alt-p';
+            assert_screen 'encrypted_volume_password_prompt';
+            type_password;
+            send_key 'ret';
```

Comment:
isn't this line is doing same as line 47 just in different way ? can we keep this consistent ?

Reply:
I would even go further . what about such refactor :

```
if (! match_has_tag 'encrypted_volume_activation_prompt-embedded_password_prompt') {
            send_key 'alt-p';
            assert_screen 'encrypted_volume_password_prompt';
        }
        type_password;
         send_key 'ret';

```

Reply:
you can replace send_key 'ret'  with send_key $cmd{ok}  it doesn't matter

Reply:
no, unfortunately it does not seem to behave the same. In the new dialog somehow there is no button pre-selected so 'ret' does not do anything because no button is focused but only the password prompt. This is why I kept it separate. I did not want to change old behaviour and did not check if 'ok' works in the old way. Can we keep it as is?